src/boss/mar/us.c

```diff
@@ -0,0 +1,284 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include <stage.h>
+
+INCLUDE_ASM("boss/mar/nonmatchings/us", func_us_8018AC0C);
```

Comment:
I notice the func_placeholder names now have _us_ inserted in them. Is that an intentional change now that will apply to future overlays? Might have just been something I missed in one of the previous PRs.

Reply:
Yep it is intentional